packages/prettier/index.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  tabWidth: 2,
+  useTabs: false,
+  semi: true,
+  singleQuote: true,
+  bracketSpacing: true,
+  arrowParens: "avoid",
```

Comment:
Think I'd go with `always` here, as when you're changing an arrow function from one param to two, you have to first wrap with parentheses and then add the param (as opposed to just adding the param), which is kinda sucky